### ISSUE TITLE
Delayed op use cases out of date?

### ISSUE THREAD

**Issue by billpratt** [capitalone/cloud-custodian#2754]
The following AWS use cases show examples of delayed operations using custom tags. When implementing these types of policies for Azure, I noticed you have to add the custom tag to the ```marked-for-op``` type. Is this the case for AWS?

http://capitalone.github.io/cloud-custodian/docs/usecases/rdsdeleteunused.html
http://capitalone.github.io/cloud-custodian/docs/usecases/elbgarbagecollection.html

**Comment by kapilt** [comment]
there is a default, the example just shows overriding it, if you have multiple workflows running against the same resource type, its important to use a separate tag per workflow to avoid them overriding each other's state.

**Comment by billpratt** [comment]
In elbgarbagecollection you are using tag  ```tag: maid_status``` The tag appears to be missing in the delete action step
```actions:
     - delete
     - type: notify
       template: default.html
       priority_header: 1
       subject: "ELB - Deleted Stale ELB - [custodian {{ account }} - {{ region }}]"
       violation_desc: "No Instances Are Attached To ELB for over 14 days:"
       action_desc: "Actions Taken:  The ELB has been deleted"
       to:
          - CloudCustodian@Company.com
          - resource-owner
       transport:
            type: sqs
            queue: https://sqs.us-east-1.amazonaws.com/12345678900/cloud-custodian-mailer
            region: us-east-1
```

**Comment by kapilt** [comment]
yeah.. `maid_status` is the default value here (set in c7n/tags.py DEFAULT_TAG). prior to being public, custodian had a different name, which is still reflected in this value and the cloud watch metrics namespace (CloudMaid), we ended up preserving those due to our compatibility guarantees.. and its kept going. probably worth updating the examples to always be explicit or at least document the default.
